logp/core.go

```diff
@@ -213,7 +213,7 @@ func createLogOutput(cfg Config, enab zapcore.LevelEnabler) (zapcore.Core, error
 	case MacOSServiceEnvironment, WindowsServiceEnvironment:
 		fallthrough
 	default:
-		return makeFileOutput(cfg, enab)
```

Comment:
This does not disable logging when it's systemd or container environment, is it intended?

Reply:
Not, but it was disabling logging when it the environment was `MacOSServiceEnvironment` or `MacOSServiceEnvironment` because of the `fallthrough. I fixed it in 7b4c5e5